local/install_deps_linux.bash

```diff
@@ -160,6 +161,7 @@ fi
 # Setup pipenv and install python dependencies.
 $PYTHON -m pip install --user pipenv
 $PYTHON -m pipenv --python $PYTHON
+$PYTHON -m pipenv lock
```

Comment:
Is this step needed? We already have a Pipfile.lock file in this repo.

Reply:
You are right; this isn't needed due to https://github.com/google/clusterfuzz/commit/59de121ad4465f15e19a47ae9e58973af12ca367, but I missed that because reproduce-tool-stable had not been updated yet.